src/loss/loss_priori.jl

```diff
@@ -98,3 +96,48 @@ function create_randloss_derivative(
         error("ERROR: Unsupported number of dimensions: $dim")
     end
 end
+
+function create_loss_priori(model, ps, st)
+    mseloss = Lux.GenericLossFunction((ŷ, y) -> abs2(ŷ - y); agg = mean)
+    ŷ = model(y, ps, st)
+end
+
+"""
+Wrap loss function `loss(batch, θ)`.
+
+The function `loss` should take inputs like `loss(f, x, y, θ)`.
+"""
+create_loss_priori(loss, f) = (f, θ, st, (x, y)) -> loss(f, θ, st, (x, y))
+
+"""
+Compute MSE between `f(x, θ, st)` and `y`.
+The MSE is further divided by `normalize(y)`.
+This signature has been chosen for compatibility with Lux.
+"""
+mean_squared_error(f, θ, st, (x, y); normalize = y -> sum(abs2, y), λ = sqrt(1e-8)) = sum(
+    abs2, f(x, θ, st)[1] - y) / normalize(y) + eltype(x)(λ) *
+                                                                                      sum(
+    abs2, θ)
```

Comment:
Yes, is still being used to create `loss_priori` (used for option 1 of training). I did however delete the `create_loss_priori` that would create a `Lux.GenericLossFunction` because that was not correct nor used.